dxr/hgext/previous_revisions.py

```diff
@@ -0,0 +1,21 @@
+"""previous_revisions
```

Comment:
No need to repeat the filename at the top. (I never understood the point of that convention. Did you?)